docs/howto/operate/move-hub.md

```diff
@@ -179,7 +179,28 @@ AzureFile needs to be mounted in the source NFS VM in order to copy the data.
    <your_scp_or_rsync_command> <homes-directory-location-on-the-source-nfs-server> /mnt/new-nfs/prod/
    ```
 
-## Transfer the JupyterHub Database
+```{note}
+If the total size of the home directories is considerable, then copying the files from one cluster to another might take a long time. So make sure you have enough time to perform this operation and check the trasnfer rates once the data transfer starts.
+
+Tip: You can use [this script](https://github.com/yuvipanda/datahub/blob/cache/scripts/rsync-active-users.py) that performs a parallel `rsync` of home directories for active users only.
```

Comment:
I wonder if we should move this script in out [`extra_scripts`](https://github.com/2i2c-org/infrastructure/tree/master/extra_scripts) directory?

Reply:
@GeorgianaElena sorry for the belated comment, but that link should be eventually updated to point to the script living now in the `extra_script` directory, right?

Reply:
You're totally right @damianavila! I forgot to update the link too.

> @GeorgianaElena sorry for the belated comment

Better late than forget about it! Thank you @damianavila :pray: Will open a PR to update it now